src/pages/Register/Register.js

```diff
@@ -1,7 +1,189 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Register.scss';
 const Register = () => {
-  return <div />;
+  // 초기값 세팅 = 아이디, 비밀번호, 비밀번호 확인, 이름, 생년원일, 휴대폰 번호
+  const [person, setPerson] = useState({
+    id: '',
+    pw: '',
+    pwConfirmed: '',
+    name: '',
+    birth: '',
+    phone: '',
+  });
+
+  const onChange = e => {
+    const { name, value } = e.target;
+    setPerson({ ...person, [name]: value });
+  };
+
+  // 아이디, 비밀번호 정규식
+  const passwordRegExp =
+    /^(?=.*[a-z])(?=.*\d)(?=.*[$@$!%*#?&])[a-z\d$@$!%*#?&]{8,16}$/;
+  const idRegExp = /^[a-z]+[a-z0-9]{4,12}$/g;
+  // input란 아래 메시지 출력
+
+  function BirthMessage() {
+    if (person.birth === '') {
+      return <p />;
+    }
+    if (person.birth.includes('.')) {
+      return <p style={{ color: 'red' }}>"1998-07-05" 형태로 입력해주세요.</p>;
+    }
+  }
+
+  function IdMessage() {
+    if (person.id === '') {
+      return <p />;
+    }
+    if (!idRegExp.test(person.id)) {
+      return (
+        <p style={{ color: 'red' }}>
+          4-12사이 대소문자 또는 숫자만 입력해 주세요!
+        </p>
+      );
+    } else {
+      return <p style={{ color: 'green' }}>사용가능한 아이디 입니다.</p>;
+    }
+  }
+
+  function PwMessage() {
+    if (person.pw === '') {
+      return <p />;
+    }
+    if (!passwordRegExp.test(person.pw)) {
+      return (
+        <div>
+          <p style={{ color: 'red' }}>
+            영어(소문자),숫자,특수문자 중 최소 2가지 이상의 문자
+          </p>
+          <p style={{ color: 'red' }}>조합 8-16자로 입력해주세요</p>
+        </div>
+      );
+    } else {
+      return <p style={{ color: 'green' }}>'안전한 비밀번호 입니다.</p>;
+    }
+  }
+
+  function PwConfirmedMessage() {
+    if (person.pwConfirmed === '') {
+      return <p />;
+    }
+    if (person.pw !== person.pwConfirmed) {
+      return <p style={{ color: 'red' }}>비밀번호가 다릅니다.</p>;
+    } else {
+      return <p style={{ color: 'green' }}>비밀번호가 같습니다.</p>;
+    }
+  }
+
+  // 통신 회원가입
+  let signUp = () => {
+    fetch('http://10.58.5.132:8000/users/signup', {
+      method: 'POST',
+      body: JSON.stringify({
+        name: person.name,
+        username: person.id,
+        password: person.pw,
+        mobile_number: person.phone,
+        birth_day: person.birth,
+      }),
+    })
+      .then(response => response.json())
+      .then(result => {});
+  };
+
+  return (
+    <div>
```

Comment:
아무런 스타일 없이 최상단을 감싸고 있는 요소라면 React.Fragment를 활용하시면 됩니다.
```suggestion
    <React.Fragment>
```
혹은
```suggestion
    <>
```